config/sidebar.config.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
         },
         {
             type: "category",
-            label: "Writing Smart Contracts",
+            label: "On Chain Development",
```

Comment:
Have you considered the title "Writing On-Chain Code"?
I know this has been debated quite a bit, and if the developers want "On-Chain Development", I will accept it. However,  the term is confusing because the development isn't an "on-chain" action. The code is sent to the chain afterward. 
```suggestion
            label: "On-Chain Development",
```

Reply:
@EdHastingsCasperLabs What do you think of this as a change? I can see where Iulia is coming from, and "Writing On-Chain Code" still solves the problem.